docs/source/index.rst

```diff
@@ -15,7 +15,7 @@
 
     tutorial/registration
     tutorial/cross_val
-    tutorial/custom
+    tutorial/run_cluster
```

Comment:
What does this have to do with modifying losses + using registry? why has this been changed in this PR?

Reply:
just correct small things together, they are minor stuff

Reply:
Minor or not, same reason as above, we lose things in the fold - please add to changelog and in future raise separate ticket?

Reply:
done